src/js/pyodide.js

```diff
@@ -198,6 +203,35 @@ export async function loadPyodide(config) {
   Module.indexURL = baseURL;
   let packageIndexReady = initializePackageIndex(baseURL);
 
+  if(config.input) {
```

Comment:
Could you factor this out into a function in `module.js` called `setStandardStreams(stdin, stdout, stderr)`, import it into this file and here just invoke `setStandardStreams(config.input, config.print, config.printErr)`? I think that's the best place to put this.

Reply:
Of course! Should it be possible to call those functions at any point during runtime or only before Pyodide gets loaded?

Reply:
I was thinking the `setStandardStreams` function should be private, only invoked in `loadPyodide`.

Reply:
Sounds good